mcs/restapi/admin_config.go

```diff
@@ -50,7 +51,8 @@ func registerConfigHandlers(api *operations.McsAPI) {
 
 // listConfig gets all configurations' names and their descriptions
 func listConfig(client MinioAdmin) ([]*models.ConfigDescription, error) {
-	configKeysHelp, err := client.helpConfigKV("", "", false)
+	ctx := context.TODO()
```

Comment:
context.TODO is deprecated use context.Background @Alevsk

Reply:
done